letsdrill.rb

```diff
@@ -1,13 +1,26 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+ if integer >= 90
+ 	"A"
+ elsif integer >= 80
+ 	"B"
+ elsif integer >= 70
+ 	"C"
+ elsif integer >= 60
+ 	"D"
+ else
+ 	"F"
+ end
 end
 
 def shortest_string(array)
-
-  #Put your code here!
-
+	# shortest = array.first
+	# array.each{|string|
+	# 	if string.length < shortest.length
+	# 		shortest = string
+	# 	end
+	# }
+	# shortest
+	shortest = array.inject {|shortest,word| word.length < shortest.length ? word : shortest}
```

Comment:
This is better, and kinda clever.  A lot of folks end up with `sort_by(&:length).first`, or better yet `min_by(&:length)`.